gravel_beta.brf

```diff
@@ -37,7 +42,9 @@ assign any_cycleway or cycleway=track|lane|shared_lane|shared or cycleway_right
 assign any_cycleroute or route_bicycle_icn=yes or route_bicycle_ncn=yes or route_bicycle_rcn=yes route_bicycle_lcn=yes
 assign is_paved highway=primary|primary_link|secondary|secondary_link|tertiary|tertiary_link|road|residential|unclassified|living_street|service|cycleway
 
+assign cycleroute_benefit 0.75

```

Comment:
Make this configurable as parameter? For example, in the zossebart profiles:

`assign mtb_hard_factor          = 2               # %mtb_hard_factor% | Difficulty of the route (0:xc/am...2:allmountain...4:enduro) | number`